examples/examples.yaml

```diff
@@ -1,6 +1,7 @@
 - path: "models/file"
   slow: ["choropleth.py"]
   no_js: ["maps_cities.py", "maps.py", "trail.py"]
+  no_diff: ["graphs.py"]
```

Comment:
Why is that? Is there anything we can do other than simply silencing this example? If there is a way, it should be preferred over no_diff or no_js.